src/System.Private.CoreLib/shared/System/Delegate.cs

```diff
@@ -0,0 +1,107 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+
+namespace System
+{
+    public abstract partial class Delegate : ICloneable, ISerializable
+    {
+		internal Delegate()
```

Comment:
Why did this ctor change from private to internal?

Reply:
I used the version from corert, it's private now